staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -69,6 +70,15 @@ var (
 		},
 		[]string{"resource"},
 	)
+	etcdLeaseObjectCounts = compbasemetrics.NewHistogramVec(
+		&compbasemetrics.HistogramOpts{
+			Name:           "etcd_lease_object_counts",
+			Help:           "Number of objects attached to a single etcd lease.",
+			Buckets:        []float64{10, 50, 100, 500, 1000, 2500, 5000},
+			StabilityLevel: compbasemetrics.ALPHA,
+		},
+		[]string{"ttl"},
```

Comment:
I'm wondering about this one - if at some point we decide to use TTL in a smarter way, we may end-up with huge histogram.
Do we really need this?

Reply:
Assume we have many small TTL lease managers and one large TTL lease manager. Small TTL leases make buckets like 10 or 50 contains large data, then we can't know how the large TTL lease manager contributes to the 10/50 buckets.
But I think if we have many lease managers with different TTLs, they may need to have a different name. The name would be a better label at that time. Currently we don't have better field to use.

Reply:
We could leave this problem here until we need to use multiple TTL lease managers.